Atmosia Asteroid Space Ruin

ZAMODA opened the pull request:
## About The Pull Request
This PR adds a new space ruin, the atmos asteroid!
It contains a small atmos setup with some engineering tools, as well as some RPD upgrades.
![image](https://user-images.githubusercontent.com/36141254/228710567-1a4c4787-9e5f-427d-9f0a-cad5276f5013.png)
### Mapping March
Ckey to receive rewards: Z Man Wit Z Plan

## Why It's Good For The Game
Space deserves some more ruins.
## Changelog
:cl:
add: Added a new space ruin, the Atmos Asteroid!
/:cl:

ZAMODA committed:
Create AtmosiaRuin.dmm

ZAMODA committed:
Update AtmosiaRuin.dmm

ZAMODA committed:
Detextures sand

ZAMODA committed:
rename

ZAMODA committed:
space.dm and proper file location

ZAMODA committed:
Update atmosastroidruin.dmm

ZAMODA committed:
Update atmosastroidruin.dmm

ZAMODA committed:
Update atmosastroidruin.dmm

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

Fikou commented on code/datums/ruins/space.dm:
```diff
@@ -356,3 +356,9 @@
+/datum/map_template/ruin/space/atmosastroidruin
+	id = "atmosastroidruin"
+	suffix = "atmosastroidruin.dmm"
+	name = "Atmos Astroid"
```
it's "Asteroid"

Fikou commented on code/datums/ruins/space.dm:
```diff
@@ -356,3 +356,9 @@
+	id = "atmosastroidruin"
+	suffix = "atmosastroidruin.dmm"
+	name = "Atmos Astroid"
+	description = "A dead atmos tech in a continuously presurizing ruin"
```
```suggestion
	description = "A dead atmos tech in a continuously pressurizing ruin."
```

ZAMODA committed:
Spelling

me me kind of no good at words

ZAMODA committed:
Merge branch 'AtmosiaRuin' of https://github.com/ZAMODA/tgstation into AtmosiaRuin

ZAMODA commented:
Spelling should be fixed!

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

san7890 commented:
This must be added to the configuration, see https://github.com/tgstation/tgstation/blob/master/.github/guides/MAPS_AND_AWAY_MISSIONS.md#configuration

ZAMODA committed:
Update spaceruinblacklist.txt

ZAMODA committed:
Merge branch 'AtmosiaRuin' of https://github.com/ZAMODA/tgstation into AtmosiaRuin

ZAMODA commented:
> This must be added to the configuration, see https://github.com/tgstation/tgstation/blob/master/.github/guides/MAPS_AND_AWAY_MISSIONS.md#configuration

Done

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
add quote to paper

someone please explain to me how to properly code paper please

ZAMODA committed:
Merge branch 'AtmosiaRuin' of https://github.com/ZAMODA/tgstation into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Unloads RCD and Atmos Suit Storage Unit

ZAMODA committed:
Spelling 2

ZAMODA committed:
Locks the door leading to the RCD

(hopefully)

ZAMODA committed:
anchors the pacman by the apc

ZAMODA committed:
Replaces one of the extinguishers with an advanced one.

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA commented:
Since I got all the issues above fixed I'll wait for any more feedback and if I don't get anything I'll un-draft this PR and set it to open again.

ZAMODA commented:
Oh God why is GitHub being dumb

ZAMODA commented:
> Looks good now, but out of curiosity....
> 
> Since the lore of it is that they suffocated themselves, have you considered making it so that the interior of the ruin spawns highly pressurized / filled with CO2?

If i knew how to change the atmos of turfs/add gases directly then i could, but at the moment the best I could do was the broken CO2 miner

ZAMODA commented:
> > Looks good now, but out of curiosity....
> > Since the lore of it is that they suffocated themselves, have you considered making it so that the interior of the ruin spawns highly pressurized / filled with CO2?
> 
> If i knew how to change the atmos of turfs/add gases directly then i could, but at the moment the best I could do was the exposed CO2 miner

wait no i might have figured it out, give me a minute

ZAMODA committed:
Adds 500(???) to all the tiles exposed to the gas miner

Also lowered the the co2 in the co2 chamber down from 50000 to 500 to match the other tiles

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

ZAMODA committed:
Merge branch 'master' into AtmosiaRuin

MMMiracles reviewed (changes requested):
I'm personally a bit iffy on the existence of every major gas miner sitting all in the same ruin since they aren't used that often in the first place due to balance concerns. If some weird meta pops up around the ruin this will probably need revisited but for now it probably wouldn't hurt to see what happens.

As for the ruin itself, mind adding something like a cyclelink_helper to that airlock so it can cycle and at least prevent large amounts of air leakage? 
![image](https://user-images.githubusercontent.com/9276171/228702226-494e3226-cf51-4ea0-9189-07f522cf724d.png)
'/obj/effect/mapping_helpers/airlock/cyclelink_helper' is the path for it, the arrows just need to face the airlock you want it to link to for proper cycling. Add that and it should be fine enough for a merge.

ZAMODA committed:
Update atmosasteroidruin.dmm

ZAMODA committed:
Merge branch 'AtmosiaRuin' of https://github.com/ZAMODA/tgstation into AtmosiaRuin

ZAMODA commented:
> As for the ruin itself, mind adding something like a cyclelink_helper to that airlock so it can cycle and at least prevent large amounts of air leakage?

Done! Hopefully that works!
